packages/firebase_auth/firebase_auth/example/test_driver/user_e2e.dart

```diff
@@ -1,898 +0,0 @@
-// ignore_for_file: require_trailing_commas
```

Comment:
Removing the tests from the `firebase_auth` from `packages/firebase_auth/firebase_auth/example/` because the source of truth should be in `tests`, right?

Reply:
Makes sense to me. Double check with @Salakar as he set it up and might have a reason why e2e tests remain in example app.